.doc_gen/metadata/services.yaml

```diff
@@ -174,6 +249,11 @@ sfn:
   short: "&SFN;"
   sort: "Step Functions"
   version: states-2016-11-23
+ssm:
+  long: "&SYSlong;"
+  short: "&SYS;"
+  sort: "SSM"
```

Comment:
SSM -> Systems Manager
(sort key is short name minus AWS/Amazon)

Reply:
I'll update the entry.

Reply:
Done.